src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native.mdx

```diff
@@ -5,47 +5,47 @@ metaDescription: 'Manage New Relic and Azure integrations with agents installed
 
 import infrastructureMonitorResources from 'images/monitored-resources.png'
 
-If your team deploys software and cloud services through Azure, our [Azure Native New Relic Service](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/newrelicinc1635200720692.newrelic_liftr_payg) lets you deploy and manage New Relic directly through the Azure Marketplace. With it, you can: 
+If your team deploys software and cloud services through Azure, our [Azure Native New Relic Service](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/newrelicinc1635200720692.newrelic_liftr_payg) lets you manage your Azure-New Relic configuration directly from your Azure account. 
 
-* Create a free New Relic account or purchase New Relic directly from your Azure Portal
-* Manage billing directly through Azure Marketplace
-* Deploy New Relic APM and infrastructure agents through an automated setup flow 
-* Manage the metrics and logs for all your resources through an interactive dashboard
+The Azure Native New Relic Service provides the following capabilities:
```

Comment:
How about something a little looser, like "With the Azure Native New Relic Service, you get all of this:"

Reply:
I love the clarity in the bullets though